### ISSUE TITLE
unable to set NLS_LANG and SQLPATH env variable

### ISSUE THREAD

**Issue by meer0h** [jenkinsci/sqlplus-script-runner-plugin#57]
Hi,
Im running project that needs to connect on two databases in single flow. They have different NLS_LANG
first:
set NLS_LANG=CROATIAN_CROATIA.EE8MSWIN1250

second:
set NLS_LANG=CROATIAN_CROATIA.AL32UTF8

Also, I'm using master script that calls nested scripts. Relative reference doesn't work in sqlplus script runner because it uses absolute path to call script  (eg. C:\Oracle\sqlplus.exe -L CO4096/********@RAZVOJ_AIX.WORLD @"C:\Program Files (x86)\Jenkins\workspace\CI-INTEGRATION-DATABASE-OBJECTS-DEV-V1\CO_LEGACY\co4096\scripts\deploy.sql")

Relative reference to scripts doesn't work (eg. @..\types\cows_tp_r_street.tpb nor @@"..\types\cows_tp_r_street.tpb") with 
message SP2-0310: unable to open file "cows_tp_r_street.sql" or SP2-0310: unable to open file "Program.sql" (because spaces in absolute path).

Basically,
solution would be to interduce new parameters similarly as "Custom TNS_ADMIN" and "Custom ORACLE_HOME":
Custom NLS_LANG
Custom SQLPATH 

that executes commands
set NLS_LANG=<value>
set SQLPATH =<value>
in same session prior to SQLplus script call.

Thanx in advance. In meantime I'll need to use "Execute windows batch command" flow step that sets needed env variables and runs SQLplus command.

**Comment by boaglio** [comment]
Done, please try version 2.0.15 .
